From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Oleg <balbes-150@yandex.ru>
Date: Wed, 27 Oct 2021 18:56:02 +0200
Subject: [ARCHEOLOGY] u-boot usb m1 (#3221)

> X-Git-Archeology: - Revision 7789fef83f10954ab442401a86a0e5a166e5db55: https://github.com/armbian/build/commit/7789fef83f10954ab442401a86a0e5a166e5db55
> X-Git-Archeology:   Date: Wed, 27 Oct 2021 18:56:02 +0200
> X-Git-Archeology:   From: Oleg <balbes-150@yandex.ru>
> X-Git-Archeology:   Subject: u-boot usb m1 (#3221)
> X-Git-Archeology:
> X-Git-Archeology: - Revision aba9e063c0cf94674f38a46e5ab0e5596d88cd9c: https://github.com/armbian/build/commit/aba9e063c0cf94674f38a46e5ab0e5596d88cd9c
> X-Git-Archeology:   Date: Wed, 27 Oct 2021 19:05:45 +0200
> X-Git-Archeology:   From: Oleg <balbes-150@yandex.ru>
> X-Git-Archeology:   Subject: add start usb Station P1 (#3220)
> X-Git-Archeology:
> X-Git-Archeology: - Revision fd33c6545bbb4bf2383fc212dd305f3b09e47fac: https://github.com/armbian/build/commit/fd33c6545bbb4bf2383fc212dd305f3b09e47fac
> X-Git-Archeology:   Date: Wed, 27 Oct 2021 20:28:50 +0200
> X-Git-Archeology:   From: Piotr Szczepanik <piter75@gmail.com>
> X-Git-Archeology:   Subject: Moved remaining u-boot rockchip64 mainline patches to new directory
> X-Git-Archeology:
> X-Git-Archeology: - Revision b74ced147675075d4bd13111dead23803b6c2267: https://github.com/armbian/build/commit/b74ced147675075d4bd13111dead23803b6c2267
> X-Git-Archeology:   Date: Fri, 08 Jul 2022 15:52:27 +0200
> X-Git-Archeology:   From: Oleg <balbes-150@yandex.ru>
> X-Git-Archeology:   Subject: media u-boot-2022.07 (#3960)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 2813365dd25e3ad110936cbf014b95b38d7090ec: https://github.com/armbian/build/commit/2813365dd25e3ad110936cbf014b95b38d7090ec
> X-Git-Archeology:   Date: Mon, 07 Nov 2022 21:29:00 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Move known non working rockhip64 boards to previous boot loader (#4392)
> X-Git-Archeology:
---
 include/configs/rockchip-common.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/rockchip-common.h b/include/configs/rockchip-common.h
index ba7061a287..5dc92373b2 100644
--- a/include/configs/rockchip-common.h
+++ b/include/configs/rockchip-common.h
@@ -55,9 +55,9 @@
 
 #ifdef CONFIG_ROCKCHIP_RK3399
 #define BOOT_TARGET_DEVICES(func) \
+	BOOT_TARGET_USB(func) \
 	BOOT_TARGET_MMC(func) \
 	BOOT_TARGET_NVME(func) \
-	BOOT_TARGET_USB(func) \
 	BOOT_TARGET_PXE(func) \
 	BOOT_TARGET_DHCP(func) \
 	BOOT_TARGET_SF(func)
-- 
Armbian

